README.md

```diff
@@ -219,7 +224,7 @@ ui5: {
 }
 ```
 
-Specific config options
+Specific config options:
```

Comment:
Here is a colon, while there is none in line 213. Is it because here there are multiple list items?

Reply:
No, I wasn't consistent here and this is because I couldn't decide what to do. Usually you only use bullet lists if there is more than one list item. So you'd usually use a colon in line 213 and the config option follows directly in the same line: 
"Specific config options: [test page]"
But in this case, the whole reading flow is less consistent, so I opted to keep the bullet list and forgot to add the colon. So it should indeed be 
Specific config options:
- [test page]